docs/explanations/modeling_extensions/surrogate/api/pysmo/pysmo_polyregression.rst

```diff
@@ -102,9 +102,12 @@ The result of the ``pysmo_surrogate.PysmoPolyTrainer`` method is a python object
 
 Confidence intervals for *PysmoPolyTrainer* models
 --------------------------------------------------------------------
-**Coming soon**
 
-The confidence intervals for the regression paramaters may be viewed using the method ``confint_regression``.
+PySMO provides the user with the capability to compute confidence intervals for the regression paramaters using the ``get_confidence_intervals`` method. This can be done by passing the result of the model training and the confidence interval value (*default=0.95*) into the ``pysmo_surrogate.PysmoPolyTrainer`` object:
+
+.. code-block:: python
+  
+  >>> conf_int = pr_trainer.get_confidence_intervals(poly_train, confidence=0.99)
```

Comment:
I like how the method call is very simple, and easy to see how it connects to the trained surrogate model. It would be useful to explain as well what the object `conf_int` looks like so users know what items to expect in the returned dictionary.

Reply:
That makes sense. Done